source/Main.hx

```diff
@@ -8,6 +8,8 @@ class Main extends Sprite
 	public function new()
 	{
 		super();
+		#if (flixel != "5.0.0")
 		addChild(new FlxGame(0, 0, PlayState));
```

Comment:
This would literally not work on the version 5.0.0 and after that, it would break.
it should be if flixel >= blablabla on the PlayState thing

Reply:
now you know why I hate flixel 5.0.0